source/Templates/TeamCitySetup/.teamcity/octopus/main/buildtypes/NetFxTestBuildType.kt

```diff
@@ -0,0 +1,18 @@
+package octopus.main.buildtypes
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nunit
+
+class NetFxTestBuildType(block: BuildType.() -> Unit) : TestBuildType({
```

Comment:
Rename to `CalamariOnlyNUnitTestBuildType` (or `CalamariOnlyTestBuildType`

Reply:
Fixed